### ISSUE TITLE
App is crashing

### ISSUE THREAD

**Issue by AmolGangadhare** [codefriar/IOSAndSalesforce#1]
App gets **crashed** immediately after running the app. It doesn't even run on physical device(13.2.3).
It is able to run on the simulator with 13.2.2 but also **crashes** when the app is run for **the first time**.

Cloned repo and followed all the steps given on the trailhead (https://trailhead.salesforce.com/content/learn/projects/mini-hack-mobile-sdk-ios/prepare-your-development-environment).

Xcode : 11.2.1
Physical device : 13.2.3
Simulator : 13.2.2

### Logs 
1. When I run the project on a physical device (iPhone 8 plus **13.2.3**)
```
dyld: Library not loaded: @rpath/SalesforceAnalytics.framework/SalesforceAnalytics
  Referenced from: /private/var/containers/Bundle/Application/33EB1C2D-165E-49A9-8DE0-750ED8C00A59/SFSDKStarter.app/SFSDKStarter
  Reason: image not found
```

2. When I run the project on a simulator (iPhone 8 plus **13.2.2**)

It crashes in SDKCore -> Classes -> Security -> SFUserAccount.m line 459.
In my opinion, this is causing due to recursively calling `createInstrumentationRecord` in `RestClient`
Still im not sure as i was not able to resolve this.
```
Thread 1: EXC_BAD_ACCESS (code=2, address=0x7ffee2e77ff0)
```

**Comment by AmolGangadhare** [comment]
@codefriar

**Comment by codefriar** [comment]
@AmolGangadhare,

Using Xcode 11.2.1, I just cloned the repo and was able to successfully build. Did you make sure to clone with recursive deps?

**Comment by codefriar** [comment]
@AmolGangadhare  I'm closing this, as I was able to duplicate your issue, but only when the git clone command *did not* include the --recurses-submodules flag, per the directions.

**Comment by AmolGangadhare** [comment]
@codefriar 

Thanks for your reply.

I've followed each and every step mentioned on trailhead. I've also included --recurses-submodules while cloning the repo. As I mentioned above that while using a simulator it is crashing in SFUserAccount as the `createInstrumentationRecord` is calling recursively and on physical device, its still getting crashed just after app launch. 

I'm still getting the crash.
